.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
   - mysql
 
 addons:
-  postgresql: "9.3"
+  postgresql: "9.4"
```

Comment:
Three things:

1. we use Postgres 9.5 for our RDS broker created instances. Is there a reason for not going with 9.5?
2. there is another commit which bumps the Postgres version in Travis in this pull request, they can probably be squashed.
3. the commit message is "Update Travis config to accomodate new tests. " - what new tests? I don't understand what that means.